source/element-kinds.md

```diff
@@ -31,14 +31,19 @@ Element parameter groups associated with this element kind are:
 - [**ReferenceChangeP**](#s:ref.change.params): Reference energy change and/or reference time correction.
 - [**TrackingP**](#s:tracking.params): Tracking parameters.
 
+Example:
+```{code} yaml
+ack1:
+  kind: ACKicker
+  length: 0.3
+  ACKickerP:
+
```

Comment:
@qianglbl 

Was the `ACKickerP` parameters list intentionally left empty in this example or is this part of the follow-up work that @DavidSagan mentioned would need to be done?